cfn-resources/stream-instance/mongodb-atlas-streaminstance.json

```diff
@@ -0,0 +1,245 @@
+{
+  "typeName": "MongoDB::Atlas::StreamInstance",
+  "description": "Returns, adds, edits, and removes stream instances.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
```

Comment:
```suggestion
  "sourceUrl": "https://github.com/mongodb/terraform-provider-mongodbatlas.git",
```

Reply:
Just wondering why this change?

Reply:
see https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html#schema-properties-sourceUrl
This field is supposed to have URL for the source of this resource

Reply:
Makes sense. Just so I fully understand, why put terraform resource link instead of the cloudformation one?

Reply:
Aah my bad, it's suppossed to be CFN repo, you're right, I copied the wrong link